src/main/java/net/earthcomputer/clientcommands/command/AliasCommand.java

```
@@ -0,0 +1,174 @@
+import com.google.gson.stream.JsonReader;
+import com.mojang.brigadier.CommandDispatcher;
+import net.earthcomputer.clientcommands.features.BrigadierRemover;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.ClientPlayNetworkHandler;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.text.LiteralText;
+import net.minecraft.util.Formatting;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.Writer;
+import java.nio.file.Path;
+import java.util.HashMap;
+
+import static com.mojang.brigadier.arguments.StringArgumentType.*;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.*;
+import static net.minecraft.server.command.CommandManager.*;
+
+public class AliasCommand {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private static final Path aliasPath = FabricLoader.getInstance().getConfigDir().resolve("clientcommands").resolve("alias_list.json");
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("calias");
+
+        dispatcher.register(literal("calias")
+            .then(literal("add")
+                .then(argument("key", string())
+                    .then(argument("command", greedyString())
+                        .executes(ctx -> addAlias(getString(ctx, "key"), getString(ctx, "command"))))))
+            .then(literal("list")
+                .executes(ctx -> listAliases()))
+            .then(literal("remove")
+                .then(argument("key", string())
+                    .executes(ctx -> removeAlias(getString(ctx, "key"))))));
+
+        HashMap<String, String> aliasMap = getAliases();
+
+        if (aliasMap == null || aliasMap.isEmpty()) {
+            sendError(new LiteralText("No aliases registered"));
```

You should probably throw an exception rather than sending an error message.